sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/TriggerBinding/HttpRequestProcessor.cs

```diff
@@ -69,7 +70,8 @@ public HttpRequestProcessor(ILogger<HttpRequestProcessor> logger)
                 SubscriptionValidationResponse validationResponse = new(){ ValidationResponse = validationCode };
                 var returnMessage = new HttpResponseMessage(HttpStatusCode.OK)
                 {
-                    Content = new StringContent(JsonConvert.SerializeObject(validationResponse))
+                    // use System.Text.Json to leverage the custom converter so that the casing is correct.
+                    Content = new StringContent(STJ.JsonSerializer.Serialize(validationResponse))
```

Comment:
Does this mean the behavior of `JsonConvert.SerializeObject(validationResponse)` differs before and after this change?

Reply:
No, previously we were using a ValidationResponse model defined in the extension library which had a JsonProperty attribute to get camel casing.

Now, we are using STJ converter which uses camel casing. I couldn't continue to use NewtonSoft _and_ use the Event Grid SDK model. It seemed worth using the SDK model here to validate that it works.

Reply:
FWIW I tested it out and both Pascal and camelCase work fine when adding a subscription, but in order to maintain the same behavior as before (and to make the tests pass) it seems better to use camel case.